src/routes/solve.js

```diff
@@ -127,7 +127,7 @@ function commonSolve (req, res, next){
     if(res.locals.params.inputs !== undefined) { //TODO: handle no inputs
       for (let [key, value] of Object.entries(res.locals.params.inputs)) {
         let param = new compute.Grasshopper.DataTree('RH_IN:'+key)
-        param.append([0], [value])
+        param.append([0], Array.isArray(value) ? value : [value])
```

Comment:
@fraguada does this change to solve.js look OK? The reasoning is that each input in `inputs` could be a single item or a list.